config.yaml

```diff
@@ -14,7 +14,19 @@ monte_carlo:
       generators_t.p_max_pu.loc[:, network.generators.carrier == "solar"]: [0.5, 2]
       # ... user can add here flexibly more features for the Monte-Carlo sampling
     options:
-      distribution: "normal" # "uniform", "normal"
-      samples: 9  # number of optimizations. Note that number of samples when using scipy has to be the square of a prime number
+      # Uniform: https://chaospy.readthedocs.io/en/master/api/chaospy.Uniform.html
+      # Normal: https://chaospy.readthedocs.io/en/master/api/chaospy.Normal.html
+      # LogNormal: https://chaospy.readthedocs.io/en/master/api/chaospy.LogNormal.html
+      # Triange: https://chaospy.readthedocs.io/en/master/api/chaospy.Triangle.html
+      # Beta: https://chaospy.readthedocs.io/en/master/api/chaospy.Beta.html
+      # Gamma: https://chaospy.readthedocs.io/en/master/api/chaospy.Gamma.html
+      distribution: "Triangle" # "Uniform", "Normal", "LogNormal", "Triangle", "Beta", "Gamma
+      # [mean, std] for Normal and LogNormal
+      # [lower_bound, upper_bound] for Uniform
+      # [lower_bound, midpoint, upper_bound] for Triangle 
+      # [alpha, beta] for Beta
+      # [shape, scale] for Gamma
+      distribution_params: [0, 0.5, 1] 
```

Comment:
I wonder what is the most convenient way to deal with setting different kinds of distributions.

For example, we have a set-up for the triangle distribution and want to make a quick run for the normal one. To me, it would feel good it would suffice to replace "Triangle" with "Normal" for that. But the question is how should we interpret `distribution_params: [0, 0.5, 1]` for that. Do you have any ideas in this regard?

Reply:
We could decide to take just two inputs with regard to the Triangular Distribution. Then internally, the midpoint will be the average of the lower and upper bounds.

Reply:
I have replaced the default to a normal distribution

Reply:
Sounds like a reasonable decision. 

My feeling is the there may be different strategies to handle inputs, but now it may be a good idea to pick one reasonable approach and stick to it keeping in mind that our goal is to obtain a minimal working prototype.

Reply:
You can take a look at the recent commit. I added an option to create a mid-value if the values supplied is just 2.